app/components/TaskList.jsx

```diff
@@ -7,7 +7,12 @@ const TaskList = ({ tasks, deleteHandler }) => {
       <ul>
         {tasks.length > 0 ? (
           tasks.map((task) => (
-            <TaskItem key={task.id} task={task} deleteHandler={deleteHandler} />
+            <TaskItem 
+              key={task.id} 
+              task={task} 
+              deleteHandler={deleteHandler} 
```

Comment:
Always for the component props, consider the handler name like: **onDelete**. it starts with **on**
example: onDelete, onSubmit.

But function name should be like: **handleOnDelete**, **handleOnSubmit**

These are just best practices.